src/Infrastructure.Dapper/Infrastructure.Dapper.csproj

```diff
@@ -6,7 +6,7 @@
 
     <ItemGroup>
       <PackageReference Include="Dapper" Version="2.0.123" />
-      <PackageReference Include="System.Data.SqlClient" Version="4.8.5" />
+      <PackageReference Include="Microsoft.Data.SqlClient" Version="5.0.1" />
```

Comment:
We should be able to get rid of this totally and rely on the peer dependency in `Core.csproj`

Reply:
For some reason I thought I wanted to keep this here for clarity of dependencies, but you're right that for efficiency's sake and the clear `Core` dependency that I should get rid of it. Good catch.